components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Extensions/MonitoredItemNotificationModelEx.cs

```diff
@@ -39,6 +39,30 @@ public static class MonitoredItemNotificationModelEx {
             }
         }
 
+        /// <summary>
+        /// Convert to monitored item notifications
+        /// </summary>
+        /// <param name="notification"></param>
+        /// <param name="monitoredItems"></param>
+        /// <returns></returns>
+        public static IEnumerable<MonitoredItemNotificationModel> ToMonitoredItemNotifications(
+            this EventNotificationList notification, IEnumerable<MonitoredItem> monitoredItems) {
+            for (var i = 0; i < notification.Events.Count; i++) {
+                var monitoredItem = monitoredItems.SingleOrDefault(
+                        m => m.ClientHandle == notification?.Events[i]?.ClientHandle);
```

Comment:
Let's remove the null propagations, they make no sense. Neither `notification` or `Events[i]` can be null in the loop. It would throw earlier and null cannot be in the list.